app/assets/scripts/map.js

```diff
@@ -483,8 +484,13 @@ function makeResponsive() {
 function singleColumnResets() {
   $('#left-column').removeClass().addClass('sixteen wide column');
   $('#right-column').removeClass().addClass('sixteen wide column');
-  $('#right-column').hide();
-  setNavSwitching();
+
+  // switched from double to single
+  // don't want to hide if a resize within single view is triggered
+  if ($('#right-column').is(':visible') && $('#left-column').is(':visible')) {
```

Comment:
this should fix the issue of resizing map while in single column view disappearing issue